apps/server/src/shared/domain/builder/domain-operation.builder.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { DomainModel } from '@shared/domain/types';
+import { ObjectId } from 'bson';
+import { DomainOperationBuilder } from '.';
+
+describe(DomainOperationBuilder.name, () => {
+	afterAll(() => {
+		jest.clearAllMocks();
+	});
+
+	it('should build generic domainOperation with all attributes', () => {
+		// Arrange
+		const domain = DomainModel.PSEUDONYMS;
+		const modifiedCount = 0;
+		const modifiedRef = [];
+		const deletedRef = [new ObjectId().toHexString(), new ObjectId().toHexString()];
+		const deletedCount = 2;
```

Comment:
Please move it to the `setup()` function.

Reply:
done